Tests/AsyncHTTPClientTests/AsyncAwaitEndToEndTests.swift

```diff
@@ -647,4 +647,47 @@ extension AsyncSequence where Element == ByteBuffer {
         }
     }
 }
+
+struct AnySendableSequence<Element>: @unchecked Sendable {
```

Comment:
We have used `AnySequence` and `AnyCollection` to force a specific overload of the `HTTPClientRequest.Body.bytes` method. The Sequence/Collection passed to these methods need to be `Sendable` now but `AnySequence` and `AnyCollection` are not `Sendable`. `AnySendableSequence` and `AnySendableCollection` are simple wrappers of their non-`Sendable` counterparts.